### PR TITLE
Redmin #60 linux_module:  Linux user management functions

### PR DESCRIPTION
**Pull request by joegen**
* Fix relative path for require
  * Fix install rule

### CODE REVIEW

File: deps/nodejs.cmake [hunk 1]
```diff
@@ -31,27 +31,22 @@ add_custom_target(${_PROJ}-install-core
...
-  DEPENDS ${_PROJ}-install-core
-  WORKING_DIRECTORY ${_SRC_DIR}
-  COMMENT "Installing ${_PROJ} PM2 process manager" )
-
-add_custom_target(${_PROJ}-install-node-gyp
-  COMMAND cd ${_SRC_DIR} && npm install node-gyp -g
-  DEPENDS ${_PROJ}-install-core
-  WORKING_DIRECTORY ${_SRC_DIR}
-  COMMENT "Installing ${_PROJ} Node Gyp Add-on compiler" )
+set(_NPM_MODULES
```
Comment (earl-jann) [inline deps/nodejs.cmake]: should we put the dependencies here? or in package.json of the module? npm installs the dependencies stated in the package.json dependencies, i think...

File: node_modules/linux/src/groupadd.js [hunk 2]
```diff
@@ -7,7 +7,8 @@
 // groupadd({'base_dir:mybasedir','myUsername'}, myCallback);
 // groupadd('-b mybasedir myUsername', myCallback);
 
-var lohikaSpawn = require('./lohika-spawn.js');
+const path = require('path');
+const lohikaSpawn = require(path.join(__dirname, 'lohika-spawn.js');
```
Comment (earl-jann) [inline node_modules/linux/src/groupadd.js]: typo error - missing ")" at end. I will fix this

Comment (joegen) [inline node_modules/linux/src/groupadd.js]: No, no.   This is my pull request.  just review it and I'll take care of the changes.   Nice catch BTW.   With the unit test broken, I wasn't really able to see this error.

File: deps/nodejs.cmake (hunk 1 above)
Comment (joegen) [inline deps/nodejs.cmake]: We can have them on both.  There is really no harm done.   Since we are not using NPM to install our modules, this is needed.

**Pull request merged**